go/vt/vtgate/engine/primitive.go

```diff
@@ -223,7 +223,7 @@ type (
 		TryStreamExecute(ctx context.Context, vcursor VCursor, bindVars map[string]*querypb.BindVariable, wantfields bool, callback func(*sqltypes.Result) error) error
 
 		// Inputs is a slice containing the inputs to this Primitive
```

Comment:
nitpick: maybe should update this and explain what the maps are about

Reply:
I don't want to run CI again, I'll update the comment in https://github.com/vitessio/vitess/pull/13859